### PR TITLE
first attempts at fwdpy11 recipe

### PR DESCRIPTION
**Pull request by molpopgen**
* [X ] I have read the guidelines above.
* [ X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by molpopgen** [comment]
@bioconda/core ready fore review.

File: recipes/fwdpy11/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+export CFLAGS="-I${PREFIX}/include"
+export CXXFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+#We must force GCC on OS X
+export CC=gcc
```
Comment (bgruening) [inline recipes/fwdpy11/build.sh]: LLVM on OSX is not possible?

File: recipes/fwdpy11/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export CXX=g++
+$PYTHON setup.py install
+
+# Add more build steps here, if they are necessary.
+
```
Comment (bgruening) [inline recipes/fwdpy11/build.sh]: You can remove these comments if you like.

File: recipes/fwdpy11/build.sh (hunk 1 above)
Comment (molpopgen) [inline recipes/fwdpy11/build.sh]: Working on it.  It seems that LLVM is getting older and older on OS X.  This code requires clang 3.8 or so on Ubuntu, which is well ahead of where OS X is, AFAIK.

File: recipes/fwdpy11/build.sh (hunk 2 above)
Comment (molpopgen) [inline recipes/fwdpy11/build.sh]: Will do.

File: recipes/fwdpy11/build.sh (hunk 1 above)
Comment (molpopgen) [inline recipes/fwdpy11/build.sh]: Let me mess around for another day.  I'm able to build on my OS X machine but not on any of the TravisCI machines with clang.  

Back to it...

**Comment by molpopgen** [comment]
@bgruening llvm build on OS X is working and I cleaned up the other file.  Merge?

**Comment by bgruening** [comment]
Thanks @molpopgen! You only need to increase the build number if this package is already available on anaconda.org

**Pull request merged**